tests/interlocked_linux_ut/interlocked_linux_ut.c

```diff
@@ -0,0 +1,833 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifdef __cplusplus
+#include <cstdlib>
+#include <cstddef>
```

Comment:
should be replaced by cstdint/stdint.c I think (we use stddef for size_t but it's already included by stdint, which has all the intX_t types).